docs/development-guide.md

```diff
@@ -34,7 +34,18 @@ it is **highly** encouraged to:
 * Fork the [ceph-csi repo](https://github.com/ceph/ceph-csi) on Github.
 * Add your fork as a git remote:
    `$ git remote add fork https://github.com/<your-github-username>/ceph-csi`
+* Set up a pre-commit hook to catch issues locally.
 
+   `$ pip install pre-commit==2.5.1`
+   `$ pre-commit install`
```

Comment:
Better, but both commands are still listed on the same line.

Reply:
Thanks... I hope it looks better now.